### PR TITLE
make the module tests run in the automated build

### PR DESCRIPTION
**Pull request by johnduffell**
I noticed that one of the tests was failing locally but there were no issues with the build.

After investigation it seemed that the build only runs "package" which doesn't exist on the modules so it's skipped.
The tsc and eslint at the root search out all files in both locations, so those checks were not skipped.

This PR updates it so that pnpm test is run at the root, which should exist on all modules as well as the handlers.

There is an efficiency issue where all the builds are doing everything, this would also mean an issue in one lambda would stop them all building, but that can be dealt with if/when it causes an issue.

### CODE REVIEW

File: modules/zuora/test/billingPreview.test.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
-import { getNextInvoiceItems } from '../src/billingPreview';
```
Comment (johnduffell) [inline modules/zuora/test/billingPreview.test.ts:1]: this was the line that was broken in the test

File: modules/jest.config.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
```
Comment (johnduffell) [inline modules/jest.config.js:1]: this and the file below are simply to make sure tests in the modules root can run

File: .github/workflows/ci-typescript.yml [hunk 3]
```diff
@@ -39,6 +39,7 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'pnpm'
       - run: pnpm install
+      - run: pushd modules && pnpm test; popd
```
Comment (rupertbates) [inline .github/workflows/ci-typescript.yml:42]: does `pnpm --filter modules test` work? If so it's more idiomatic

**Review by rupertbates — APPROVED** [review]

File: .github/workflows/ci-typescript.yml (hunk 3 above)
Comment (johnduffell) [reply .github/workflows/ci-typescript.yml]: no
```
Run pnpm --filter modules test
No projects matched the filters in "/home/runner/work/support-service-lambdas/support-service-lambdas"
```

**Pull request merged**